cypress/e2e/signup.cy.js

```diff
@@ -9,18 +9,24 @@ describe('Signup', () => {
     cy.findByRole('button', { text: 'Submit' }).click();
     cy.get('.error').should('not.exist');
     cy.get('.error_message').should('not.exist');
+
   });
 
   describe('form validation displays an error when', () => {
-    xit('the username is empty', () => {
+    it('the username is empty', () => {
+      cy.findByLabelText('Username').type(' ');
```

Comment:
Question: ' ' vs. ''

Reply:
Thanks for the feedback Jay. cy.type() doesn't accept an empty string so that's why I was going to use cy.clear() after type() but I must've missed it here. I think we can also use cy.invoke with passing an empty string value as well. In this case I was going to use clear() so that's why I didn't use .trim()